web/src/components/Navigation/ProfileInfo/ProfileInfo.tsx

```diff
@@ -67,22 +77,25 @@ const ProfileIcon: React.FC = () => {
     <Popover>
       {({ isOpen, close: closePopover }) => (
         <React.Fragment>
-          <PopoverTrigger
-            as={AbstractButton}
-            display="flex"
-            width={40}
-            height={40}
-            backgroundColor={isOpen ? 'violet-400' : 'violet-500'}
-            _hover={{ backgroundColor: 'violet-400' }}
-            transition="background-color 0.1s linear"
-            borderRadius="circle"
-            justifyContent="center"
-            alignItems="center"
-            fontSize="small"
-            fontWeight="bold"
-            aria-label="Toggle User Menu"
-          >
-            {getUserInitials(userInfo).toUpperCase()}
+          <PopoverTrigger as={AbstractButton} aria-label="Toggle User Menu">
```

Comment:
Make sure we limit its content to the sidebar's width (to protect against overflows on user full names)

```suggestion
          <PopoverTrigger as={AbstractButton} aria-label="Toggle User Menu" width="100%">
```